### PR TITLE
Upgrade office-ui-fabric-react to 5.29.0

### PR DESCRIPTION
**Pull request by chrismohr**
This is in preparation to implement TextField, since we'll need suffix from https://github.com/OfficeDev/office-ui-fabric-react/pull/3348

There were 3 changes to work around:
- Persona root size height is now calculated to accommodate a border indicating presence (https://github.com/OfficeDev/office-ui-fabric-react/pull/3291)
- Red is no longer in the PersonaCoin color palette (https://github.com/OfficeDev/office-ui-fabric-react/pull/3249)
- DirectionHint keys are no longer available in JS (https://github.com/OfficeDev/office-ui-fabric-react/pull/2984#pullrequestreview-66299384)

### CODE REVIEW

**Comment by chrismohr** [comment]
I think we need to look closer at `config/styleguide-visual/reference/Callout_alignments_desktop.png`

It's slightly different now, however the original is not as I would have expected.

**Comment by chrismohr** [comment]
@unindented, @swese44 please take a look.

**Comment by swese44** [comment]
i think we can just remove the Callout Alignments example and its visual diff reference image, what do you think @unindented?

File: src/components/Avatar/Avatar.md [hunk 1]
```diff
@@ -225,53 +225,53 @@ const { AvatarSize } = require('.');
...
-      <Avatar size={AvatarSize.MEDIUM} name="EEE EEE" initials="OR" />
+      <Avatar size={AvatarSize.MEDIUM} name="M" initials="PU" />
     </FixedGridColumn>
     <FixedGridColumn>
-      <Avatar size={AvatarSize.MEDIUM} name="FFF FFF" initials="DR" />
+      <Avatar size={AvatarSize.MEDIUM} name="N" initials="BL" />
     </FixedGridColumn>
     <FixedGridColumn>
-      <Avatar size={AvatarSize.MEDIUM} name="HHH HHH" initials="RE" />
+      <Avatar size={AvatarSize.MEDIUM} name="O" initials="TE" />
```
Comment (swese44) [inline src/components/Avatar/Avatar.md]: if i'm reading the Fabric PR correctly, they've removed RED and gone from 15 colors to 14. Can we update this example to remove the 15th avatar (duplicate color)?

File: src/components/Callout/Callout.md [hunk 2]
```diff
@@ -3,7 +3,6 @@
 Default hover trigger:
 
 ```js { "props": { "data-example": "default hover trigger" } }
-const { DirectionalHint } = require('.');
```
Comment (swese44) [inline src/components/Callout/Callout.md]: is this not working anymore?

Comment (unindented) [reply src/components/Callout/Callout.md]: @chrismohr points it out in the description of the PR. They've gone from normal `enum`s to `const enum`s, which apparently don't generate JavaScript. They're citing reduced bundle size as the reason. So now we cannot use their constants from JavaScript!

Comment (unindented) [reply src/components/Callout/Callout.md]: As a solution, I was thinking of using the `updateExample` property, extracting all examples to their own separate files, and compiling them with `tsc`. That'd also solve the issue you cited of not being able to use your IDE to write examples. Whatchuthink?

Comment (swese44) [reply src/components/Callout/Callout.md]: @unindented are you suggesting that as part of this PR or as a follow-up item?

Comment (unindented) [reply src/components/Callout/Callout.md]: Let's say followup. I don't want this to be blocked.

Comment (chrismohr) [reply src/components/Callout/Callout.md]: https://github.com/Microsoft/YamUI/issues/131

**Pull request merged**